detect_secrets/plugins/private_key.py

```diff
@@ -1,68 +1,25 @@
-"""
-This code was extracted in part from
```

Comment:
I'd recommend we keep this, better safe than sorry IMO https://github.com/Yelp/detect-secrets/issues/31

Reply:
Alright.